contexts/gs1-8110-coupon/v1.json

```diff
@@ -31,4 +32,4 @@
       }
     }
   }
-}
+}
```

Comment:
I think it's more common to configure your editor to have a trailing newline.  There will be churn on this if everyone is not doing the same thing.  This effects many files in this commit.  It will also change the bytes and hash of a context, which could be a caching issue.

Reply:
Hopefully we're only hashing the canonical JSON--and not the raw bytes. Additionally, I've only ever had trailing whitespace removed...so it's news to me that it's more common. 😁 That said, let me know what's preferred here...and we can setup tooling to keep us all on the same road. https://editorconfig.org/ is nice. 🐭